### DISCUSSION TITLE
Adding tags and Mentions

### DISCUSSION THREAD

**Discussion by techker** [MicrosoftPremier/VstsExtensions#206] [Q&A] [answered]
Hey my task is set as
- task: CreateWorkItem@1
  inputs:
    workItemType: 'Task'
    title: 'New Build is Done for Android TV : $(Build.BuildNumber)'
    assignedTo: 'Myemail
    areaPath: 'Main\MyTeam'
    iterationPath: 'Main'
    fieldMappings: |
      Description=Pipeline-$(Build.BuildNumber) see attached files and work items from Develop branch
      Developed by=Me
      Verified By=Qa
      Custom.BuildVersionCode=$(Build.BuildNumber)
    associate: true
    linkWorkItems: true
    linkType: 'Tasks'
    linkTarget: 'associate'
    addAttachments: true
    attachments: '**/*.apk'
    createOutputs: true
    outputVariables: |
      TaskCreatedID=System.Id
      WorkItem=CWI.WorkItemEditUrl
 
Is there a way to add Tags?

And in description if i add @user it does not add it as a notification Mention filed. it sets it as a string.

**Comment by ReneSchumacher** [comment, marked as answer]
Hi @techker,

you can add tags through the `fieldMappings` property just like any other field. Just specify the field name Tags or System.Tags and set the value to a semicolon-delimited list of tags (e.g., `System.Tags=Tag1;Tag2;Tag2`).

Mentions are on our roadmap. Unfortunately, the logic to create a proper mention tag is done by the work item UI, so it's not enough to just put `@username` into the description. A correct mention looks like this: `<a href=\"#\" data-vss-mention=\"version:2.0,ae0cd6b4-e6e0-484b-a42a-d96ffca86e26\">@René Schumacher</a>` and requires you to know the VSID of the user you want to mention. I'm pretty busy right now, but I will try to add the mention logic to the task soon.

**Comment by techker** [comment]
perfect thx!
